src/MCT.js

```diff
@@ -83,7 +83,7 @@ define([
          * @memberof module:openmct.MCT#
          * @name conductor
          */
-        this.conductor = new api.TimeConductor();
+        this.time = new api.Time();
```

Comment:
I prefer "conductor" as it better matches the contents of the API, but time is so succinct.  Hrm.